CMakeLists.txt

```
@@ -212,10 +212,12 @@ if(SDLSOUND_BUILD_STATIC)
     if(NOT MSVC)
         set_target_properties(SDL2_sound-static PROPERTIES OUTPUT_NAME "SDL2_sound")
     endif()
-    target_include_directories(SDL2_sound-static PUBLIC $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}/SDL2>)
+    target_include_directories(SDL2_sound-static PUBLIC 
+        $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}/SDL2>
```

Let's also allow `#include "SDL2/SDL_sound.h"` for installed prefix'es.
This method of inclusion is currently also supported by SDL3.

```suggestion
        $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}>
        $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}/SDL2>
```

---

Done with f0b29fc0ea8c96da4198bf7adf75279acf43ad2c